ingestfile

```diff
@@ -68,28 +62,171 @@ _remove_mediaid_from_queue(){
+    _email_delivery "${MAKEBROADCAST_DELIVERY_EMAIL_TO}"
+    _report -dt "STATUS Uploading ${OBJECTSDIR}/service/${MEDIAID%.*}_SLATE.mov to the OMNEON."
+    "${SCRIPTDIR}/uploadomneon" "${OBJECTSDIR}/service/${MEDIAID%.*}_SLATE.mov"
+    _email_delivery "${MAKEBROADCAST_DELIVERY_EMAIL_TO}"
+}
+
+_delivertoaipstorage(){
+    if [ "${AIP_STORAGE}" != "${OUTDIR_INGESTFILE}" ] ; then
+    "${SCRIPTDIR}/migratefiles" -r -o "${AIP_STORAGE}" "${OUTDIR_INGESTFILE}/${MEDIAID}"
+    RSYNC_ERR="${?}"
+    [ $(du -s "${OUTDIR_INGESTFILE}/${MEDIAID}" | awk '{print $1}') = "0" ] && rm -r "${OUTDIR_INGESTFILE}/${MEDIAID}"
+    _report -dt "STATUS Done. Final package has been delivered to ${AIP_STORAGE}/${MEDIAID}"
+    else echo "As Target directory is same as Input directory, files were not migrated."
+    fi
+}
+
+_delivertoia(){
+    if [ $(echo "${MEDIAID}" | grep -E "${REGEX4IA}") ] ; then
+        if [ $(which uploadia) ] ; then
+            _report -dt "Attempting to run uploadia and upload to Internet Archive"
+            uploadia "${OUTDIR_INGESTFILE}/${MEDIAID}/objects/access/youtube_up/${MEDIAID}.mp4" "${OUTDIR_INGESTFILE}/${MEDIAID}/objects/access/dvd/${MEDIAID}.iso"
+        fi
+    fi
+}
+
+_writecapturelog(){
+    echo -e "${LOG}" >> "${INGESTLOG}"
+    _writeingestlog "media ID" "${MEDIAID}"
+    _writeingestlog "media ID slate" "${MEDIAID4SLATE}"
+    _writeingestlog "operator" "${OP}"
+    _writeingestlog "input" "${INPUT}"
+    _writeingestlog "processing_directory" "${OUTDIR_INGESTFILE}"
+    _writeingestlog "audiodecision" "${AUDIODECISION}"
+    _writeingestlog "cropdecision" "${CROPDECISION}"
+    _writeingestlog "cleanupdecision" "${CLEANUPDECISION}"
+    _writeingestlog "formula" "${FORMULA}"
+    _writeingestlog "ingest_process_dir" "${OUTDIR_INGESTFILE}"
+    _writeingestlog "ingest_process_dir_freespace" "$(df -h "${OUTDIR_INGESTFILE}" | tail -n1 | awk '{print $4}')"
+    _writeingestlog "computer_name" "$(uname -n)"
+    _writeingestlog "user_name" "$(whoami)"
+    _writeingestlog "operating_system_version" "$(uname -v)"
+    _writeingestlog "datetime_start" "$(_get_iso8601)"
+    _writeingestlog "Series Title" "${SERIESTITLE}"
+    _writeingestlog "Episode Title" "${EPISODETITLE}"
+    _writeingestlog "made this" "ingestfile"
+    _writeingestlog "Interlacement test" "${INTERLACETEST}"
+    if [ -n "$(cat "${PHASEREPORT}" | xargs)" ] ; then
+    	_writeingestlog "Out_of_phase_audio_ranges" "$(cat "${PHASEREPORT}" | xargs)"
+    else
+        _writeingestlog "Out_of_phase_audio_ranges" "None"
+    fi
+    if [ "${INTIME}" ] ; then
+        _writeingestlog "intime" "${INTIME}"
+    fi
+    if [ "${OUTTIME}" ] ; then
+        _writeingestlog "outtime" "${OUTTIME}"
+    fi
+    if [ "${MODE}" == "preservation" ] ; then
+        _writeingestlog "package type" "Preservation-Video"
+    else
+        _writeingestlog "package type" "Production"
+    fi
+}
+
 _cleanup(){
-    _log -a "Process aborted"
+    _log -a "Process canceled"
```

Comment:
seems to mix `cancel` and `abandon` as replacement language, could we use 1 rather than 2.

Reply:
Oops, yes, thanks for catching

Reply:
Changed in my other open pull request in commit d9ad5f50585c14c73a696a1d2530e97a58b732bc